rules/aip0131/http_uri_name_test.go

```diff
@@ -15,59 +15,41 @@
 package aip0131
 
 import (
-	"strings"
 	"testing"
 
-	"github.com/golang/protobuf/proto"
-	dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
-	"github.com/jhump/protoreflect/desc/builder"
-	"google.golang.org/genproto/googleapis/api/annotations"
+	"github.com/googleapis/api-linter/rules/internal/testutils"
 )
 
 func TestHttpNameField(t *testing.T) {
 	tests := []struct {
 		testName   string
-		uri        string
-		methodName string
-		msg        string
+		URI        string
+		MethodName string
+		problems   testutils.Problems
 	}{
-		{"Valid", "/v1/{name=publishers/*/books/*}", "GetBook", ""},
-		{"InvalidVarName", "/v1/{book=publishers/*/books/*}", "GetBook", "`name` field"},
-		{"NoVarName", "/v1/publishers/*/books/*", "GetBook", "`name` field"},
-		{"Irrelevant", "/v1/{book=publishers/*/books/*}", "AcquireBook", ""},
+		{"Valid", "/v1/{name=publishers/*/books/*}", "GetBook", testutils.Problems{}},
```

Comment:
Can you add a false positive case here?

Reply:
The Valid case was the false positive case. The rule and test were both broken (because the regex was messed up).